### PR TITLE
feat: Add a `roller tx fund-faucet` command

### PR DESCRIPTION
**Pull request by ItayLevyOfficial**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [x]  Targeted PR against correct branch
- [x]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x]  Linked to Github issue with discussion and accepted design
- [x]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
 
       - name: Register the rollapp
         run: |
-          go run . register --home $ROLLER_CONFIG_PATH --no-output
+          go run . tx register --home $ROLLER_CONFIG_PATH --no-output
```
Comment (mtsitrin) [inline .github/workflows/e2e.yml:54]: will require update the docs

Comment (ItayLevyOfficial) [reply .github/workflows/e2e.yml:54]: told @shaolin-flow

File: cmd/tx/fund_faucet/fund_faucet.go [hunk 2]
```diff
@@ -9,12 +9,22 @@ import (
...
 )
 
+var flagNames = struct {
+	tokenAmount string
+}{
+	tokenAmount: "token-amount",
+}
+
+const faucetAddr = "dym1g8sf7w4cz5gtupa6y62h3q6a4gjv37pgefnpt5"
+const faucetDefaultTokenAmount = "5000000"
```
Comment (mtsitrin) [inline cmd/tx/fund_faucet/fund_faucet.go:24]: it supposed to be 500 if u multiply it by decimals, no?

Comment (ItayLevyOfficial) [reply cmd/tx/fund_faucet/fund_faucet.go:24]: The [docs](https://docs.dymension.xyz/build/quick-start/roller-quick/ibc-transfer/) defaults to 5,000,000 with 18 zeros. WDYT @shaolin-flow

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**